Extensions/satsukimous.js

```diff
@@ -52,7 +67,7 @@ XKit.extensions.satsukimous = new Object({
 		}
 		$( "img" ).filter(function( index ) {
 			return $( this ).attr( "src" ).indexOf( "anonymous_avatar" ) !== -1;
-		}).attr( "src", replacement ).addClass("satsukimous_src matoiRYUKOOOOoO");
+		}).attr( "src", replacement ).addClass("satsukimous_src matoiRYUKOOOOoO").parent().parent().find(".asker > .name").text(XKit.extensions.satsukimous.preferences.replace_name.value ? XKit.extensions.satsukimous.preferences.name_replacement.value : "anonymous");
```

Comment:
Nitpick: place your addition on the next line, mostly because I don't want to scroll all the way to the right to debug